cmake/setup.py.cmake

```diff
@@ -31,18 +31,19 @@ if(BUILD_PYTHON)
   configure_file("${SETUP_PY_IN}" "${SETUP_PY}")
 
   if(PYTHONINTERP_FOUND)
-    # python setup.py build
-    add_custom_command(OUTPUT "${SETUP_PY_INIT}"
-      COMMAND "${CMAKE_COMMAND}" -E make_directory "${PYTHON_DEST}/sirf"
-      COMMAND "${CMAKE_COMMAND}" -E touch "${SETUP_PY_INIT}"
-      COMMAND "${PYTHON_EXECUTABLE}" setup.py build
-      DEPENDS "${SETUP_PY}"
-      WORKING_DIRECTORY "${PYTHON_DEST}")
+    if("${PYTHON_STRATEGY}" STREQUAL "SETUP_PY")
+      # python setup.py build
+      add_custom_command(OUTPUT "${SETUP_PY_INIT}"
```

Comment:
This seems a mixture of PYTHONPATH and SETUP_PY strategies?

Reply:
You think? It looks to be just calling the setup.py